RPG Wizardry/Assets/Scripts/Avatar/Combat/CastingManager.cs

```diff
@@ -0,0 +1,86 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using nl.SWEG.RPGWizardry.PlayerInput;
+using UnityEngine;
+
+namespace nl.SWEG.RPGWizardry.Avatar.Combat
+{
+    [RequireComponent(typeof(InputState))]
+    public class CastingManager : MonoBehaviour
+    {
+        #region Variables
+        #region Public
+        /// <summary>
+        /// Prototype projectile; fill this with selected spell later
+        /// </summary>
+        public Projectile projectilePrefab;
```

Comment:
is public set required?

Reply:
to set the projectile that should be fired, yes, i think so